proto/beacon/p2p/v1/messages.proto

```diff
@@ -106,4 +106,8 @@ message AttestationRecord {
 message CrosslinkRecord {
   uint64 dynasty = 1;
   bytes blockhash = 2;
+}
+
+message BlockRegistry {
+  repeated bytes blockhashes = 1;
```

Comment:
I believe we use `BlockHash` as nomenclature throughout the repo instead of `Blockhash`, which this proto def will compile into. I'd rather we use `repeated bytes block_hashes` to be consistent with naming elsewhere.